Documentation/Run_Code_Locally.md

```diff
@@ -1,32 +1,56 @@
-To run code locally there will be a couple files you will have to change from Release 1
+To run code locally there are several programs that must be installed
+* NPM
+* Angular cli
+* Docker
+* Mongo
+* Gradle
+* NPM Dependencies
+Installation guides for these programs are below
 
+Once everything is installed, follow these instructions in order to run the project locally
+
+Start Mongo
+* Open a command prompt and use the command `mongo`
+  * If this doesn't work, you may need to open a seperate command prompt and type the command `mongod` then reattempt the previous step
+
+Start the backend
+From Compile.io (top level directory)
```

Comment:
Looks like you need another newline here if you want to put this on another line.

Reply:
Likely you want to move this with the line above and kick the capital on the "from".